resources/dev-setup-task.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: dev-namespace-setup
```

Comment:
we need to coordinate the changes, the task reference here still mentions `rhtap-dev-namespace-setup` https://github.com/redhat-ai-dev/ai-lab-app/blob/main/templates/http/base/initialize-namespace.yaml#L36

Reply:
I can loop that change into https://issues.redhat.com/browse/DEVAI-168 as part of the switch from rhtap installer to our installer 

cc @maysunfaisal @michael-valdron

Reply:
sure @Jdubrick

Reply:
I know the changes that have to be made since I made them to test this task out to produce my screenshots above anyway so it won't be any issue